docs/ckecli.md

```diff
@@ -255,3 +246,14 @@ Set the query variables to search machines in sabakan.
 ### `ckecli sabakan get-variables`
 
 Get the query variables to search machines in sabakan.
+
+## `ckecli status`
+
+Report the internal status of the CKE server.
+
+See [schema.md](schema.md#status)
+
+Example:
+```json
+{"status":"completed","last-updated" : "2019-10-02-00:00:00"}
```

Comment:
This is not the serialization format of `time.Time` for JSON.
It should be formatted as RFC3339.